src/test/kotlin/com/terraformation/backend/tracking/db/T0PlotStoreTest.kt

```diff
@@ -265,22 +264,28 @@ internal class T0PlotStoreTest : DatabaseTest(), RunsAsDatabaseUser {
   }
 
   @Nested
-  inner class AssignT0PlotSpeciesDensity {
+  inner class AssignT0PlotSpeciesDensities {
```

Comment:
Worth having a test case for updating existing densities where the new value is missing a species that was originally there? Not sure what the desired behavior is in that case. I think the current implementation will leave the old value as-is, which is fine if that's what we want, but a test case would make it clear that it's an intentional choice.

Reply:
Good thought. I think we'll actually want to assume that the list is everything that should be in the plot and delete existing ones in the monitoring plot.